pkg/permissions/actions.go

```diff
@@ -16,6 +16,7 @@ var (
 const (
 	ReadRepositoryAction                      = "fs:ReadRepository"
 	CreateRepositoryAction                    = "fs:CreateRepository"
+	UpdateRepositoryAction                    = "fs:UpdateRepository"
```

Comment:
Tricky this one - first I agree with this solution.

My throghts
- Do we want ManageMetadata action?
- Using `Write` vs `Update`?
- Do we want two actions - one for write and one for delete?

(_Note:_ this will require migration)

Reply:
> Do we want ManageMetadata action?
It's a good suggestion, I think update repository is enough 
> Using Write vs Update?
Prefer update because it updates the repository metadata
> Do we want two actions - one for write and one for delete?
IMO leaving update is fine because the action is on the repository level, and even if you choose to change to metadata, it's still on the level of all metadata and not an object (a specific row of metadata)
Don't think it will require a migration for two reasons:
1. It's an internal API 
2. Admins already have FSFullAccess which include `fs:*`